src/TBuffer.cpp

```diff
@@ -2400,7 +2400,11 @@ void TBuffer::log( int from, int to )
                 }
                 else
                 {
-                    toLog = lineBuffer[i];
+                    if( !timeBuffer[i].isEmpty() )
```

Comment:
Can't recall the exact details of the `timeBuffer` but is it empty on wrapped lines or filled with something other than a date-time stamp - possibly some '-'? It may be that we will need to insert 13 spaces to pad things out...